go.mod

```diff
@@ -98,16 +92,36 @@ require (
 	golang.org/x/sys v0.0.0-20190410235845-0ad05ae3009d
 	golang.org/x/text v0.3.1-0.20190410012825-f4905fbd45b6
 	google.golang.org/grpc v1.20.0
-	k8s.io/api v0.0.0-20190409092523-d687e77c8ae9
-	k8s.io/apiextensions-apiserver v0.0.0-20190409093611-d42ba5c63e88 // indirect
-	k8s.io/apimachinery v0.0.0-20190409092423-760d1845f48b
-	k8s.io/client-go v0.0.0-20190409092706-ca8df85b1798
-	k8s.io/cloud-provider v0.0.0-20190409133700-fca63a37790d // indirect
-	k8s.io/kube-openapi v0.0.0-20190306001800-15615b16d372 // indirect
-	k8s.io/kubernetes v1.14.0
-	k8s.io/utils v0.0.0-20190308190857-21c4ce38f2a7
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v0.0.0
+	k8s.io/cri-api v0.0.0
+	k8s.io/kubernetes v0.0.0
+	k8s.io/utils v0.0.0-20190221042446-c2654d5206da
 )
 
-replace cloud.google.com/go => cloud.google.com/go v0.34.0
-
-module github.com/cri-o/cri-o
+replace (
+	cloud.google.com/go => cloud.google.com/go v0.34.0
+	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v0.1.2-0.20190408193819-a1b50f621a48
```

Comment:
I am not sure those need to be replaced. Can't we keep them in the require section only?

Reply:
Unfortunaetly not, because k8s directly uses v1.0.0 which is too old for our use case: https://github.com/kubernetes/kubernetes/blob/master/go.mod#L121